sa3ed_app/www/lost_person_search/index.js

```diff
@@ -1,23 +1,44 @@
 function get_lost_person_list(search_criteria={}){
     let args = {};
-    
-    //TODO: map search_criteria to args
-    run_api(method="sa3ed_app.api.LostPersonEndPoints.get_lost_persons",
-                    type= "GET".
-                    async = false,
-                    args = search_criteria,
-                    function_render_response = render_response_get_lost_person_list
-            );
-}
-
-function render_response_get_lost_person_list(response){
-    const label = document.getElementById("div_test_response");
-    label.innerHTML = response['error_message'];
+    for (const key in search_criteria) {
+        if (search_criteria.hasOwnProperty(key)) {
+            args[key] = search_criteria[key];
+        }
+    }
+    frappe.call({
+        method: "sa3ed_app.api.LostPersonEndPoints.get_lost_persons",
+        args: {args_obj:args},
+        callback: function(result) {
+            var container = document.getElementById("person_conatiner");
```

Comment:
here to avoid any Run time issues with JS you need to check if the element is added to the DOM before using it.

Reply:
This issue should be solved by Commit 8dc005d